how-to-release.md

```diff
@@ -0,0 +1,274 @@
+Each release should be executed by a Release Manager, who is selected/proposed by the steering committee. This document describes the process that the Release Manager follows to perform a release. Any changes to this process should be discussed and adopted on the dev@ mailing list.
+
+This release process should be strictly followed by Pravega and also followed by all the ecosystem projects inside Pravega organization by default, but Pravega ecosystem projects can have its own release instruction page that points to this one as a reference and then explains what is different.
+
+If you are releasing the project, then you must read and follow the instructions carefully. The releases are a public face of the project and most users interact with the project only through the releases, so they need to be treated with great importance.
+
+# Overview
+
+![release process](https://cwiki.apache.org/confluence/download/attachments/73631092/release-guide-1.png)
+
+The release process consists of several steps:
+
+1. Decide to release
+2. Prepare for the release
+3. Build a release candidate
+4. Vote on the release candidate
+5. If necessary, fix any issues and go back to step 3.
+6. Finalize the release
+7. Promote the release
+
+As the current projects have dependencies within these projects, the formal release should happen in the following order.
+
+> Pravega -> Pravega Keycloak Client / Pravega Spark Connector -> Pravega Schema Registry -> Pravega Flink Connector -> Pravega Samples
+
+# Decide to release
+Deciding to release is the first step of the release process. This is a consensus-based decision of the entire community, There is no formal process to decide to release, but any objections needs to be resolved by consensus before starting the release.
```

Comment:
where do this discussion happen ? on slack ? on the ML ?

Reply:
Just added the announcement in `#dev` on Slack